src/Teams/Data/IRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Teams.Data
+{
+    interface IRepository<T> where T : class
+    {
+        IQueryable<T> GetAll();
+        Task <T> GetById { get; set;}
```

Comment:
Why space between Task and T?
Where Async in the end?
How can I send string Id for Team when will go to implement repository?